### PR TITLE
[feature-wip](MTMV) Sync finish status only for tasks

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Issue Number: close #xxx

mtmv task keep only finish status,reduce the loss caused by logging

After changes, unfinished tasks will be directly lost when fe master restart

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run p0

**Comment by zddr** [comment]
run buildall

**Review by adonis0147 — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVTaskManager.java [hunk 1]
```diff
@@ -328,86 +318,10 @@ public MTMVTask getTask(String taskId) throws AnalysisException {
...
-                status.setMessage(changeTask.getErrorMessage());
-                status.setErrorCode(changeTask.getErrorCode());
-                status.setState(toStatus);
-                status.setFinishTime(changeTask.getFinishTime());
-                addHistory(status);
-            }
-        } else {
-            LOG.warn("Illegal  Task taskId:{} status transform from {} to {}", changeTask.getTaskId(), fromStatus,
-                    toStatus);
+        if (MTMVUtils.getNowTimeStamp() > task.getExpireTime()) {
```
Comment (SaintBacchus) [inline fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVTaskManager.java:321]: clear expired task in reply. it will cause inconsinst in different fe.

**Comment by adonis0147** [comment]
run buildall

**Review by adonis0147 — APPROVED** [review]
LGTM

**Pull request merged**